### DISCUSSION TITLE
SVG-in-JS 리팩터링 하기

### DISCUSSION THREAD

**Discussion by hozzijeong** [woowacourse-teams/2023-pium#315] [뜨거운 토론]
제가 [어느 글](https://github.com/yeonjuan/dev-blog/blob/master/JavaScript/breaking-up-with-svg-in-js-in-2023.md)을 보고 과연 피움 서비스는 이미지 최적화가 잘 되어있을까? 라는 의문 점을 갖고 있었습니다.

그래서 실제로 **Lighthouse** 분석을 통해 저희 서비스의 성능을 확인해 봤는데 결과는 다음과 같습니다. (이미지는 내 식물 상세보기 입니다)

![image](https://github.com/woowacourse-teams/2023-pium/assets/50974359/e20f0e8f-da77-4f0c-8c97-765634e0a410)

다른 페에지도 확인해보면 퍼포먼스가 전부 떨어지는 것을 볼 수 있습니다.

이미지나 아이콘 최적화를 위해서 SVG-in-JS 방식이 아닌 아래 이미지에 따라 아이콘 최적화를 해보는것은 어떤가요?

![image](https://github.com/woowacourse-teams/2023-pium/assets/50974359/4be394a3-8dcb-40b0-9937-1f2a8ad3f686)

**Comment by WaiNaat** [comment]
완 (svg 스프라이트 적용)
#355
